### ISSUE TITLE
Being able to remove a filter in the search using the Backspace key.

### ISSUE THREAD

**Issue by Nico-Kestra** [kestra-io/kestra#6404]
### Feature description

It’s all in the title! :-)

**Comment by MilosPaunovic** [comment]
You can do that already @Nico-Kestra.

**Comment by Nico-Kestra** [comment]
Ah yes, indeed, for the ones I add, but not for the default one (absolute date) when I first arrive on the page.

**Comment by satvik2131** [comment]
Hi @Nico-Kestra , I guess I am able to do it , correct me if it's the wrong thing .

https://github.com/user-attachments/assets/46a6cf86-f307-415a-a641-c411dbe02b1c

**Comment by Piyush-r-bhaskar** [comment]
> Hi @Nico-Kestra , I guess I am able to do it , correct me if it's the wrong thing .
> 
> https://github.com/user-attachments/assets/46a6cf86-f307-415a-a641-c411dbe02b1c
> 
> 

@satvik2131 could you add more filters and check how backspace is working in Dashboard only ?

**Comment by Piyush-r-bhaskar** [comment]
> > Hi @Nico-Kestra , I guess I am able to do it , correct me if it's the wrong thing .
> > 
> > https://github.com/user-attachments/assets/46a6cf86-f307-415a-a641-c411dbe02b1c
> > 
> > 
> 
> @satvik2131 could you add more filters and check how backspace is working ?

**Comment by satvik2131** [comment]
> > Hi @Nico-Kestra , I guess I am able to do it , correct me if it's the wrong thing .
> > 
> >   
> >     
> >     
> > 
> >     Screencast.From.2024-12-14.19-04-37.mp4
> 
> @satvik2131 could you add more filters and check how backspace is working in Dashboard only ?


https://github.com/user-attachments/assets/2be9215d-d1a0-49fe-9c13-d2308f3b55fb

Like this ?

**Comment by Piyush-r-bhaskar** [comment]
> > > Hi @Nico-Kestra , I guess I am able to do it , correct me if it's the wrong thing .
> > > 
> > >   
> > >     
> > >     
> > > 
> > >     Screencast.From.2024-12-14.19-04-37.mp4
> > 
> > @satvik2131 could you add more filters and check how backspace is working in Dashboard only ?
> 
> 
> https://github.com/user-attachments/assets/2be9215d-d1a0-49fe-9c13-d2308f3b55fb
> 
> Like this ? 

Yes..but 3-4 filters.

**Comment by satvik2131** [comment]
> > > > Hi @Nico-Kestra , I guess I am able to do it , correct me if it's the wrong thing .
> > > > ```
> > > > Screencast.From.2024-12-14.19-04-37.mp4
> > > > ```
> > > 
> > > 
> > > @satvik2131 could you add more filters and check how backspace is working in Dashboard only ?
> > 
> > 
> > 
> >   
> >     
> >     
> > 
> >     Screencast.From.2024-12-15.13-24-01.mp4
> >     
> >   
> > 
> >   
> > 
> >   
> > 
> > 
> > Like this ?
> 
> Yes..but 3-4 filters.


https://github.com/user-attachments/assets/00c79bfb-06bf-4137-bd84-435214888750

I have added several filters as well , if this is also not the case , can you share a screen recording showing the issue .

**Comment by Piyush-r-bhaskar** [comment]
> > > > > Hi @Nico-Kestra , I guess I am able to do it , correct me if it's the wrong thing .
> > > > > ```
> > > > > Screencast.From.2024-12-14.19-04-37.mp4
> > > > > ```
> > > > 
> > > > 
> > > > @satvik2131 could you add more filters and check how backspace is working in Dashboard only ?
> > > 
> > > 
> > > ```
> > > Screencast.From.2024-12-15.13-24-01.mp4
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > Like this ?
> > 
> > 
> > Yes..but 3-4 filters.
> 
>  Screencast.From.2024-12-15.13-36-38.mp4 
> I have added several filters as well , if this is also not the case , can you share a screen recording showing the issue .

Perfect...issue was not able to remove for the default one (absolute date). & it's looks good as per recording.
why don't you raise a PR for @MilosPaunovic to review and merge.

**Comment by satvik2131** [comment]
> > > > > > Hi @Nico-Kestra , I guess I am able to do it , correct me if it's the wrong thing .
> > > > > > ```
> > > > > > Screencast.From.2024-12-14.19-04-37.mp4
> > > > > > ```
> > > > > 
> > > > > 
> > > > > @satvik2131 could you add more filters and check how backspace is working in Dashboard only ?
> > > > 
> > > > 
> > > > ```
> > > > Screencast.From.2024-12-15.13-24-01.mp4
> > > > ```
> > > > 
> > > > 
> > > >     
> > > >       
> > > >     
> > > > 
> > > >       
> > > >     
> > > > 
> > > >     
> > > >   
> > > > Like this ?
> > > 
> > > 
> > > Yes..but 3-4 filters.
> > 
> > 
> > Screencast.From.2024-12-15.13-36-38.mp4
> > I have added several filters as well , if this is also not the case , can you share a screen recording showing the issue .
> 
> Perfect...issue was not able to remove for the default one (absolute date). & it's looks good as per recording. why don't you raise a PR for @MilosPaunovic to review and merge.




> > > > > > Hi @Nico-Kestra , I guess I am able to do it , correct me if it's the wrong thing .
> > > > > > ```
> > > > > > Screencast.From.2024-12-14.19-04-37.mp4
> > > > > > ```
> > > > > 
> > > > > 
> > > > > @satvik2131 could you add more filters and check how backspace is working in Dashboard only ?
> > > > 
> > > > 
> > > > ```
> > > > Screencast.From.2024-12-15.13-24-01.mp4
> > > > ```
> > > > 
> > > > 
> > > >     
> > > >       
> > > >     
> > > > 
> > > >       
> > > >     
> > > > 
> > > >     
> > > >   
> > > > Like this ?
> > > 
> > > 
> > > Yes..but 3-4 filters.
> > 
> > 
> > Screencast.From.2024-12-15.13-36-38.mp4
> > I have added several filters as well , if this is also not the case , can you share a screen recording showing the issue .
> 
> Perfect...issue was not able to remove for the default one (absolute date). & it's looks good as per recording. why don't you raise a PR for @MilosPaunovic to review and merge.

I have not made any changes , it's default

**Comment by Piyush-r-bhaskar** [comment]
> > > > > > > Hi @Nico-Kestra , I guess I am able to do it , correct me if it's the wrong thing .
> > > > > > > ```
> > > > > > > Screencast.From.2024-12-14.19-04-37.mp4
> > > > > > > ```
> > > > > > 
> > > > > > 
> > > > > > @satvik2131 could you add more filters and check how backspace is working in Dashboard only ?
> > > > > 
> > > > > 
> > > > > ```
> > > > > Screencast.From.2024-12-15.13-24-01.mp4
> > > > > ```
> > > > > 
> > > > > 
> > > > >     
> > > > >       
> > > > >     
> > > > > 
> > > > >       
> > > > >     
> > > > > 
> > > > >     
> > > > >   
> > > > > Like this ?
> > > > 
> > > > 
> > > > Yes..but 3-4 filters.
> > > 
> > > 
> > > Screencast.From.2024-12-15.13-36-38.mp4
> > > I have added several filters as well , if this is also not the case , can you share a screen recording showing the issue .
> > 
> > 
> > Perfect...issue was not able to remove for the default one (absolute date). & it's looks good as per recording. why don't you raise a PR for @MilosPaunovic to review and merge.
> 
> > > > > > > Hi @Nico-Kestra , I guess I am able to do it , correct me if it's the wrong thing .
> > > > > > > ```
> > > > > > > Screencast.From.2024-12-14.19-04-37.mp4
> > > > > > > ```
> > > > > > 
> > > > > > 
> > > > > > @satvik2131 could you add more filters and check how backspace is working in Dashboard only ?
> > > > > 
> > > > > 
> > > > > ```
> > > > > Screencast.From.2024-12-15.13-24-01.mp4
> > > > > ```
> > > > > 
> > > > > 
> > > > >     
> > > > >       
> > > > >     
> > > > > 
> > > > >       
> > > > >     
> > > > > 
> > > > >     
> > > > >   
> > > > > Like this ?
> > > > 
> > > > 
> > > > Yes..but 3-4 filters.
> > > 
> > > 
> > > Screencast.From.2024-12-15.13-36-38.mp4
> > > I have added several filters as well , if this is also not the case , can you share a screen recording showing the issue .
> > 
> > 
> > Perfect...issue was not able to remove for the default one (absolute date). & it's looks good as per recording. why don't you raise a PR for @MilosPaunovic to review and merge.
> 
> I have not made any changes , it's default

Yes, we can do backspace and remove for those added manually already but here issue was only for default filter selected as it was not being removed on backspace Now it should be resolved from your change.

**Comment by satvik2131** [comment]
@Piyush-r-bhaskar , actually , I have not made any changes  , I guess it would have fixed in any recent commits , can you try switching to develop branch and check if the issue still there ?

**Comment by Piyush-r-bhaskar** [comment]
https://github.com/user-attachments/assets/94a67040-ae26-4037-b907-64103ef1c421

For me issue is still there with updated `develop` branch.

**Comment by satvik2131** [comment]
yup got it , just took latest pull and getting the issue , will check the cause .

**Comment by satvik2131** [comment]
@Piyush-r-bhaskar and @MilosPaunovic , I have added a fix , please check the PR and let me know if there is some issue

**Comment by Ben8t** [comment]
@satvik2131 @MilosPaunovic any update on this?

https://github.com/kestra-io/kestra/compare/develop...satvik2131:kestra:fix(ui)-dashboard-filter-remove-default

**Comment by satvik2131** [comment]
@Ben8t  , actually I was stucked on some part . The issue is the e-select uses v-model in the sample code , where removing with backspace works . And when we trigger removeItem() , it didn't get triggered for the first time , but when we remove the last item by clicking the cross button and then add filters and then removes them , it works fine . I am unable to debug the issue , let me know if anyone can help me .

**Comment by satvik2131** [comment]
https://github.com/user-attachments/assets/fbf7d96e-653c-47c1-9b16-182736f50bf4
the issue is this , I am not able to debug the issue , please let me know if anyone can guide me here .
Like when the pages loads at first and we try to remove the last tag , it won't get removed , 
but after removing it manually with the remove button , it get's removed and then adding more items in filter and removing all also works fine 

I am picking other issue until then 
@MilosPaunovic

**Comment by MilosPaunovic** [comment]
Yeah, you can take your pick from our [curated list of good first issues](https://go.kestra.io/contributing) to see if there is something you find interesting enough and think you can tackle independently.

**Comment by MilosPaunovic** [comment]
Closing this as it does not seem relevant anymore. To be re-opened if necessary.

**Comment by ReLoutre** [comment]
Still unable to remove filters by using the backspace key.

Environment:
Instance: [Postgres OSS Preview](https://postgres-oss.preview.dev.kestra.io/)

Version: Latest (published at 02:45 AM today)

https://jam.dev/c/9b630f3c-1fd5-49ea-94c4-99cadef878d7

**Comment by ReLoutre** [comment]
Making a new issue instead of reopening this one : https://github.com/kestra-io/kestra/issues/8036
